src/components/Sidebar/index.js

```diff
@@ -1,21 +1,44 @@
-import React, { useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 
-import userImage from 'images/user-placeholder.jpg';
+import ProfileAdapter from 'adapters/profileAdapter';
+import { AuthContext } from 'contexts/auth';
 
 const Sidebar = () => {
   const [sidebar, setSidebar] = useState(false);
+  const [name, setName] = useState(false);
+  const [avatar, setAvatar] = useState(false);
   const showSidebar = () => setSidebar(!sidebar);
+  const authContext = useContext(AuthContext);
 
   const logout = () => {
     localStorage.clear();
 
     window.location.href = '/login';
   };
 
+  useEffect(() => {
+    getProfile();
+  }, []);
+
+  const getProfile = async () => {
+    try {
+      await ProfileAdapter.get(authContext.state.authorization_token).then(
+        function (response) {
+          if (response.status === 200) {
+            setName(response.data.data.type);
+            setAvatar(response.data.data.attributes.avatar_url);
+          }
+        }
+      );
+    } catch (error) {
+      console.log(error);
```

Comment:
@rafayet-monon What's the case that we get and error?
Is that the case about the the `authorization_token` is expired?

Reply:
@Lahphim Not really. If some error occurs in that specific API.